### DISCUSSION TITLE
MicroPython on a new WFI mcu

### DISCUSSION THREAD

**Discussion by joeygold** [micropython/micropython#11842] [General]
[github.com/MicrochipTech/PIC32MZW1_MicroPython](url)

Limited but a start.

**Comment by massimosala** [comment]
Clicking on the link... 404 page not found

**Reply by mattytrentini** [reply]
The text is fine, the href is not. Corrected link: [https://github.com/MicrochipTech/PIC32MZW1_MicroPython](https://github.com/MicrochipTech/PIC32MZW1_MicroPython)

**Reply by joeygold** [reply]
github.com/MicrochipTech/PIC32MZW1_MicroPython
